docs/4.0.0/rules/index.md

```diff
@@ -0,0 +1,117 @@
+---
```

Comment:
Is this file just a copy of `docs/rules/index.md`? If so, could it be replaced with a symlink?